Source/RimConnection/About/Manifest.xml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
What's this compared to the `About.xml`?

Reply:
This is just for Fluffy's Mod Manager, https://github.com/fluffy-mods/ModManager/blob/master/ForModders.md, so we can show version in their mod menu